libs/langchain/langchain/retrievers/self_query/base.py

```diff
@@ -94,6 +94,7 @@ def _get_builtin_translator(vectorstore: VectorStore) -> Visitor:
     from langchain_community.vectorstores import (
         Pinecone as CommunityPinecone,
     )
+    from langchain_weaviate.vectorstores import WeaviateVectorStore
```

Comment:
can we conditionally import the way we do with elasticsearch in L141

Reply:
Sure

Reply:
Done